src/EnergyPlus/HVACControllers.cc

```diff
@@ -369,13 +371,10 @@ void ManageControllers(EnergyPlusData &state,
         // Simulate the correct Controller with the current ControlNum
         ControllerType = ControllerProps(ControlNum).ControllerType_Num;
 
-        {
```

Comment:
I think this select case var is better suited to just IF/ELSE, as it will 99.9999999% of the time just be the if side, and it's very easy to read.

Reply:
If there are only two cases, use `if-else`.